packages/calcite-components/src/components/action/action.tsx

```diff
@@ -95,6 +95,13 @@ export class Action extends LitElement implements InteractiveComponent {
   /** Use this property to override individual strings used by the component. */
   @property() messageOverrides?: typeof this.messages._overrides;
 
+  /**
+   * When `true`, the component is full width
+   *
+   * @private
+   */
+  @property({ reflect: true }) fullWidth = false;
```

Comment:
I wonder if we should just make this a public property and have the user be in control of setting it? Thoughts? What if we just made it a public property like `stretch` or `grow`? @macandcheese thoughts?